src/main/kotlin/no/nav/tilleggsstonader/sak/statistikk/vedtak/domene/StønadstypeDvh.kt

```diff
@@ -14,6 +16,8 @@ enum class StønadstypeDvh {
                 Stønadstype.BARNETILSYN -> BARNETILSYN
                 Stønadstype.LÆREMIDLER -> LÆREMIDLER
                 Stønadstype.BOUTGIFTER -> BOUTGIFTER
+                Stønadstype.DAGLIG_REISE_TSO -> DAGLIG_REISE_TSO
+                Stønadstype.DAGLIG_REISE_TSR -> DAGLIG_REISE_TSR
```

Comment:
Bør kanskje vente med dette til vi har hørt hva datavarehus forventer å få her?
Da kan typene også fjernes fra `StønadstypeDvh`

```suggestion
                Stønadstype.DAGLIG_REISE_TSO -> TODO("Daglig reise er ikke implementert enda")
                Stønadstype.DAGLIG_REISE_TSR -> TODO("Daglig reise er ikke implementert enda")
```